docs/documentation/upgrading/topics/changes/changes-26_2_0.adoc

```diff
@@ -48,6 +48,46 @@ If your deployment scripts add explicit NetworkPolicies for {project_name}, you
+the {project_name} had only preview token exchange feature, which is now referred as link:{securing_apps_token_exchange_link}#_legacy-token-exchange[Legacy token exchange] (Feature `token-exchange:v1`).
+The legacy token exchange still remains preview and it works the same way as in previous releases. In case you used the link:{securing_apps_token_exchange_link}#_internal-token-to-internal-token-exchange[internal-internal token exchange],
+it is recommended to migrate to the new standard token exchange instead.
+
+If needed for some reason, you can stick to the legacy token exchange and it should work the same way as in previous releases. For continue using legacy token exchange, it is not needed to disable the standard token exchange feature
+as your clients will use standard token exchange just if it is explicitly enabled on the {project_name} client as described below. However migration to the standard token exchange is recommended as that is
+officially supported and will be the priority for the {project_name} team in the future.
+
+Here are some notes, which can help to migrate to the new standard token exchange:
+
+* The feature `token-exchange-standard`, which represents the new Standard token exchange, is enabled by default. It is recommended to just
+disable the `token-exchange` feature, which represents the Legacy token exchange, to make sure that requests will be served by the new standard token exchange.
+
+* It is possible to have both standard and legacy token exchange features enabled, which can be useful if you need to cover standard use cases (internal-internal) together with the other token exchange use cases,
+which are implemented only by legacy token exchange. Like for instance link:{securing_apps_token_exchange_link}#_external-token-to-internal-token-exchange[external to internal token exchange] is implemented only by
+legacy token exchange. In this case {project_name} will serve standard internal-to-internal requests preferably by the standard token exchange when the other requests will be served by the legacy token exchange. It is detected based on the
+parameters if the particular request is standard or not. For instance requests containing non-standard parameters like `requested_issuer` or `requested_subject` are considered legacy.
+
+* Standard token exchange requires to enable switch on the client as described in the link:{securing_apps_token_exchange_link}#_standard-token-exchange-enable[{securing_apps_token_exchange_name}].
+
+Few more clarifications regarding the notable changes in the behaviour of standard token exchange in comparison to legacy token exchange:
+
+* Fine-grained admin permissions are no longer needed and no longer supported for the standard token exchange.
+
+* The most notable change regarding the behaviour of scopes and audiences is, that applied client scopes are based on the client triggering the token exchange request, rather than "target" client specified by the `audience` parameter.
```

Comment:
```suggestion
* The most notable change regarding the behavior of scopes and audiences is that the applied client scopes are based on the client triggering the token exchange request rather than the "target" client specified by the `audience` parameter.
```